frontend/src/components/pages/Scheduling/VolunteerInformation.tsx

```diff
@@ -13,13 +25,124 @@ const VolunteerInformation = ({
 }: SchedulingStepProps) => {
   const { previous, next } = navigation;
   const history = useHistory();
-  //   const { insert form fields for this page here } = formData;
+  const {
+    startTime,
+    endTime,
+    frequency,
+    volunteerNeeded,
+    volunteerTime,
+    pickupLocation,
+    isPickup,
+    notes,
+  } = formValues;
+
+  const volunteerNeededValues = ["Yes", "No"];
+  const volunteerAssistanceValues = [
+    "Pickup (food rescue)",
+    "Unloading (on site)",
+  ];
+  const volunteerRequiredHelperText =
+    "A volunteer is a community fridge member who will assist with donation dropoffs. Information of the volunteer assigned will be provided.";
+
+  const handleChange = (e: boolean | string, name: string) => {
+    setForm({ target: { name, value: e } });
+  };
+
+  const volunteerNeededRadioValue = () => {
```

Comment:
had to make these instead of using ternary operators in the components themselves because we don't allow nested ternary operators in our project D':